nncf/quantization/algo.py

```diff
@@ -1140,8 +1140,8 @@ def weight_qp_filter_fn(qp_id_):
             if not graph_node:
                 raise RuntimeError(f'Internal error: failed to find node for by scope={module_scope}')
 
-            op_arch = target_model.get_op_arch_by_graph_node(graph_node)
-            is_adjust_padding_applicable = op_arch in adjust_padding_operation_set
+            op_type = graph_node[NNCFGraph.OP_TYPE_NODE_ATTR]
```

Comment:
Will separately refactor accesses such as this one to work with NNCFNode's instead of the underlying nx dict.